linkedevents/tests/test_remap_events.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import tempfile
+
+import pytest
+from django.core.management import call_command
+
+
+@pytest.mark.parametrize("apply", [True, False])
+@pytest.mark.django_db
+def test_remap_events(place, place2, event, apply):
+    with tempfile.TemporaryDirectory() as d:
```

Comment:
I'm a pytest n00b so I'll try it both as a learning exercise and because that'd make it easier to avoid duplicated code if the test is split in two completely separate test cases.